docs/Dockerfile

```diff
@@ -75,12 +75,6 @@ WORKDIR /rootski/docs
 COPY ./docs/requirements.txt ./requirements.txt
 RUN pip install -r requirements.txt
 
-
-# change to a new user because drawio-desktop raises an error when run as root
```

Comment:
This is no longer needed Just checking cause it looks like it was important for drawio that was just implemented.

Reply:
I'll clean this up in a future PR involving sphinx. We should have many.